canonical-web-and-design/ubuntu-com-security-api
# Add read all cve and notices

## Done

Read CVEs and Notices endpoints.

## QA
- Clone repo & fetch changes
- docker-compose up -d
- `dotrun`
- Browse to http://0.0.0.0:8030/docs you should see the documentation with the two new endpoints. 
The endpoings cannot be tested unless there is a local database.
